File: celsius_conversion/python_celsius_conversion/celsius_conversion.py
#comment
import sys
import math

def toFahrenheit ( x ):
  return  x * (9.0/5.0) + 32.0

def toCelsius ( x ):
  return (x - 32.0) * (5.0/9.0)

def main():
  if len(sys.argv) != 1:
    print ("Usage: %s <noargs>" % sys.argv[0])
    sys.exit()
  print("toCelsius(70.0)=<" + str(toCelsius(70.0)) + ">")
  print("toFahrenheit(21.0)=<" + str(toFahrenheit(21.0)) + ">")
  sys.exit(), end

main()

File: celsius_conversion/javascript_celsius_conversion/celsius_conversion.js
function toCelsius(x) {
  return (5.0/9.0) * (x - 32.0);
}

function toFahrenheit(x) {
  return  x * (9.0/5.0) + 32.0;
}

function main() {
  console.log("toCelsius(" + 70 + ")=<" + toCelsius(70.0) + ">");
  console.log("toFahrenheit(" + 21 + ")=<" + toFahrenheit(21.0) + ">");
}

main()
